### PR TITLE
Add an initial Payline documentation

### PR DESCRIPTION
**Pull request by real34**
This PR adds the current way to install Payline (early preview) in a Front-Commerce app.

I also fixed a minor mistake in Affirm's documentation.

[Preview :arrow_upper_right: ](https://deploy-preview-165--heuristic-almeida-1a1f35.netlify.app/docs/advanced/payments/payline.html)

### CODE REVIEW

File: source/docs/advanced/payments/payline.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+<blockquote class="feature--new">
+  _This feature has been added in versions: Front-Commerce `2.5.0`_
+</blockquote>
+
+<blockquote class="note">
+**Note** This integration is aimed at being transparent for administrators and developers. That is why we haven't duplicated documentation from existing Magento resources. Please [contact us](mailto:contact@front-commerce.com) if you need further assistance.
+</blockquote>
+
+Front-Commerce for Magento 1 requires a `FrontCommerce_Payline` module that turns the [Monext Payline official Magento 1 extension](https://docs.monext.fr/display/DT/Plugin+Magento+1) into a headless payment method for Front-Commerce. It is aimed at being transparent for Magento administrators and developers while allowing for a better Customer experience in a Front-Commerce application.
```
Comment (JulienPradet) [inline source/docs/advanced/payments/payline.md:23]: I'd add a link to the FrontCommerce_Payline module here (even if the installation is described below).

File: source/docs/advanced/payments/payline.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+This integration is slightly different from [traditional Magento1 headless payments](/docs/magento2/headless-payments.html) in that sense that it contains a "companion component" in Front-Commerce. The component allows to get payment information from the checkout page. No redirection to Payline is involved (except for 3DS or PayPal authentication), the Customer remains on the Front-Commerce store.
```
Comment (JulienPradet) [inline source/docs/advanced/payments/payline.md:25]: The text of the link mentions M1, but the link is M2. Not sure which one you wanted here.

File: source/docs/advanced/payments/payline.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
++      connectSrc: ["*.payline.com"],
++      fontSrc: ["*.cdn.payline.com", "maxcdn.bootstrapcdn.com"],
+      baseUri: []
+    }
+  },
+```
+
+### That's it!
+
+You can now configure the Magento extension to use a HOMO mode and [test the integration](https://docs.monext.fr/display/DT/Les+cartes+de+test)
```
Comment (JulienPradet) [inline source/docs/advanced/payments/payline.md:114]: `HOMO` mode? It'd be worth mentioning where this configuration is in M1.

**Review by JulienPradet — APPROVED** [review]
Thanks for this documentation :)

File: source/docs/advanced/payments/payline.md (hunk 2 above)
Comment (real34) [reply source/docs/advanced/payments/payline.md:25]: The M1 indeed :disappointed_relieved: 
Thanks!

File: source/docs/advanced/payments/payline.md (hunk 3 above)
Comment (real34) [reply source/docs/advanced/payments/payline.md:114]: Yes, good idea. I added a link to https://docs.monext.fr/display/DT/Plugin+Magento+1#PluginMagento1-Commonsettings/Param%C3%A8trescommuns

"HOMO" is the codename for "HOMOLOGATION" but it's named HOMO… while the alternative is "PRODUCTION" :man_shrugging:

**Pull request merged**